### PR TITLE
Improve Italian translation

### PR DESCRIPTION
**Pull request by Fale**
As title

### CODE REVIEW

**Comment by sftim** [comment]
BTW polishment? I think you mean “polishing”. Although I'd avoid using the word “polish” when talking about localizations lest it get confused for _Język polski_!

File: content/it/case-studies/box/index.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+title: Box Case Study
```
Comment (sftim) [inline content/it/case-studies/box/index.html:2]: nit: not translated

Comment (sftim) [reply content/it/case-studies/box/index.html:2]: (nor is the rest of the page)

**Comment by Fale** [comment]
oops, good point :-D. I'm going to change it, thanks!

File: content/it/case-studies/box/index.html (hunk 1 above)
Comment (Fale) [reply content/it/case-studies/box/index.html:2]: Yeah, I pulled it so the Box block appears in the case studies page. Being a copy of the English one, it does not change the rendering if not adding the box clock

File: content/it/community/_index.html [hunk 2]
```diff
@@ -146,7 +146,7 @@ <h1 style="padding-top:1%">Discussioni</h1>
 github&nbsp;&#9654;
 </a>
 <div class="resourceboxtext" style="font-size:12px;text-transform:none !important;font-weight:300;line-height:1.4em;color:#333333;margin-top:4%">
-Tracciamento di tutti i progetti e issue, e ovviante il codice stesso
+Tutti i problemi, issue, e ovviamente il codice stesso
```
Comment (fabriziopandini) [inline content/it/community/_index.html:149]:
```suggestion
Tutti i progetti, le issue, e ovviamente il codice stesso
```

File: content/it/training/_index.html [hunk 3]
```diff
@@ -85,7 +85,7 @@ <h2>Certificati Kubernetes</h2>
           <h5>
             <b>Certified Kubernetes Application Developer (CKAD)</b>
           </h5>
-          <p>L'esame "Certified Kubernetes Application Developer" certifica che la persona è in grado di progettare, costruire, configurare, ed esporre un applicativo Cloud Native su Kubernetes.</p>
+          <p>L'esame "Certified Kubernetes Application Developer" certifica che la persona è in grado di progettare, costruire, configurare, e riesciare in produzione un applicativo Cloud Native su Kubernetes.</p>
```
Comment (fabriziopandini) [inline content/it/training/_index.html:88]:
```suggestion
          <p>L'esame "Certified Kubernetes Application Developer" certifica che la persona è in grado di progettare, costruire, configurare, e rilasciare in produzione un applicativo Cloud Native su Kubernetes.</p>
```

**Comment by Fale** [comment]
Thanks @fabriziopandini, fixed them :-)

**Comment by jnardiello** [comment]
LGTM :) Well done @Fale

**Comment by micheleberardi** [comment]
LGTM Great Job @Fale

**Comment by fabriziopandini** [comment]
/hold
waiting for some late feedback to get merged

**Comment by fabriziopandini** [comment]
@fale thanks for the quick reaction!
/approve
/lgtm

**Comment by sftim** [comment]
Happy to see this hit the branch
/approve

**Pull request merged**